python/tests/transformers/named_image_test.py

```diff
@@ -140,9 +143,9 @@ def test_DeepImagePredictor(self):
         kerasPredict = self.kerasPredict
         transformer = DeepImagePredictor(inputCol='image', modelName=self.name,
                                          outputCol="prediction",)
-        fullPredict = transformer.transform(self.imageDF).collect()
+        fileOrder = self.fileOrder
+        fullPredict = sorted(transformer.transform(self.imageDF).collect(),key=lambda x:fileOrder[x['image']['origin'].split('/')[-1]])
```

Comment:
Suggestion: Put this sorting into a helper function since it appears multiple times and/or add a comment explaining what it does?

Reply:
Yup, both suggestions make sense.